### PR TITLE
Improvements to node template README 

### PR DESCRIPTION
**Pull request by sacha-l**
This PR fixes a few broken links to the Substrate docs and makes a few clarifications in the README of the node-template. It also updates the link to the `construct_runtime` macro to the latest rustdoc Substrate release (previously pointing [here](https://crates.parity.io/frame_support/macro.construct_runtime.html)).

### CODE REVIEW

File: bin/node-template/README.md [hunk 1]
```diff
@@ -125,27 +125,28 @@ Take special note of the following:
...
+The runtime in this project is constructed using many FRAME pallets that ship with [the Substrate repository](https://github.com/paritytech/substrate/tree/master/frame) and a template pallet that is [defined in the `pallets`](./pallets/template/src/lib.rs) directory.
 
-A FRAME pallet is compromised of a number of blockchain primitives:
+A FRAME pallet is comprised of a number of blockchain primitives, including:
 
 - Storage: FRAME defines a rich set of powerful [storage abstractions](https://docs.substrate.io/build/runtime-storage/) that makes it easy to use Substrate's efficient key-value database to manage the evolving state of a blockchain.
 - Dispatchables: FRAME pallets define special types of functions that can be invoked (dispatched) from outside of the runtime in order to update its state.
-- Events: Substrate uses [events and errors](https://docs.substrate.io/build/events-and-errors/) to notify users of important changes in the runtime.
+- Events: Substrate uses [events](https://docs.substrate.io/build/events-and-errors/) to notify users of runtime updates.
```
Comment (bkchr) [inline bin/node-template/README.md:146]: IMO `updates` sounds too generic and could be anything, like a runtime update :P

Comment (gilescope) [reply bin/node-template/README.md:146]: to notify users of "significant changes of state" ?

Comment (sacha-l) [inline bin/node-template/README.md:146]:
```suggestion
- Events: Substrate uses [events](https://docs.substrate.io/build/events-and-errors/) to notify users of significant state changes.
```

Comment (sacha-l) [reply bin/node-template/README.md]: Updated this to use more accurate language. Good catch.

**Review by bkchr — APPROVED** [review]

**Review by dmitry-markin — APPROVED** [review]

**Comment by sacha-l** [comment]
bot merge

**Pull request merged**